index.html

```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
 </div>
 
  <div class="card-works">
+  <header class="about-me"> 
+    <h2> About myself</h2>
+  </header>
+   
+
+   <p class="abt-p">
+    Hello I’m a software developer! I can help you build a product , feature or website. 
+    Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+   </p>
+
+   <h2 class="lets-connect">let's connect</h2>
+
+   <div class="link">
+    <ul class="media-icons">
+      <li>
+        <a href="#">
+          <img src="./images/Twitter.png" alt="twitter">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Linkedin icon.png" alt="linkedin">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Mail.png" alt="mail">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Github.png" alt="github">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img src="./images/Vector.png" alt="vector">
+
+  
+
+  </div>
+    <h1 class="resume">Get my resume</h1>
+  
+     <div class="langs">
+      <h1 class="lang1"> languages</h1>
+      
+      <ul class="languages">
+        <li> <a href="#"> <img src="./images/Frame 9.png" alt=""></a> </li>
+        <li> <a href="#"><img src="./images/Frame 10.png" alt=""></a> </li>
+        <li> <a href="#"><img src="./images/Frame 11.png" alt=""></a></li>
+      </ul>
+      <div class="frameworks">
+      <h1 class="frameworks"> Frameworks</h1>
```

Comment:
- [ ] I'll suggest you use `h2 or h3` tag here as you are meant to have only 1 h1 in your webpage and that should be the main title. :+1: :smile: